Automattic/wp-calypso
# Add close button to Theme Showcase Banner

This PR adds a close button to the Theme Showcase Banner.

Related: #23944

- - -

Testing Instructions:

1) Go to My Sites > Themes
2) Click on the [ x ] on the Showcase Banner

Expected Behavior: Showcase banner should close, and the closed state should persist across reloads (where the state is not hydrated).

Actual Behavior: Showcase banner closes, but reappears when reloading (when state is not hydrated).
